frontend/src/token-store.ts

```diff
@@ -262,6 +262,17 @@ class Token {
     isNativeToken: true,
   };
 
+  static gaiaChainUSDC: CoinMeta = {
```

Comment:
Looks like this one is the same as `CoinList`

Reply:
We will have to refactor them both. We must have only one.

Reply:
Removed